dallinger/config.py

```diff
@@ -269,6 +270,17 @@ def load(self):
     def register_extra_parameters(self):
         initialize_experiment_package(os.getcwd())
         extra_parameters = None
+
+        # Import and instantiate the experiment class if available
+        # This will run any experiment specific parameter registrations
+        from dallinger.experiment import load
+
+        try:
+            exp_klass = load()
+            exp_klass()
+        except ImportError:
```

Comment:
I've had problems with this pattern in the past, where bad imports in `experiment.py` (e.g. package not installed) end up getting muffled. But maybe this is why we always need to do `dallinger verify` first! In any case, I don't think this is the only place where this happens in Dallinger, so it's not like this is introducing a new problem.